conditions_test.go

```diff
@@ -46,6 +46,11 @@ func TestAddressUnmarshalJSON(t *testing.T) {
 			json:     `"cond:foo/bar/636f6e646974696f6e64617461"`,
 			wantAddr: weave.NewCondition("foo", "bar", []byte("conditiondata")).Address(),
 		},
+		"bech32 decoding": {
+			// $ bech32  -e -h tiov 746573742d7061796c6f6164
+			json:     `"bech32:tiov1w3jhxapdwpshjmr0v9jqymqq4y"`,
+			wantAddr: weave.Address("test-payload"),
```

Comment:
This is actually an invalid address... they must be 20 bytes.
Test is fine, but maybe update test vector so the address is valid, and some extra assertions in our code for more correctness don't break this test later

Reply:
Done in https://github.com/iov-one/weave/pull/415/commits/8c2c36251bf16de923a06da8fddd673c1923b4b9